tool/before-install.sh

```diff
@@ -23,6 +23,8 @@ fi
 
 ./tool/shared/install-dart-sdk.sh
 
+travis_fold() { true; }
```

Comment:
All definitions need to be conditional otherwise you'll (soft) break things for Travis CI.

Reply:
Thanks for the review. 
Fixed all definitions.